openfisca_france/model/revenus/remplacement/indemnites_journalieres_securite_sociale.py

```diff
@@ -74,3 +73,9 @@ def function(self, simulation, period):
         )
 
         return period, result
+
+class date_arret_de_travail(Variable):
+    column = DateCol(default = nan)
```

Comment:
@fpagnoux as you can see in the traceback, the crash is in `/variables` endpoint.